packages/cli/src/commands/org/search/dump.ts

```diff
@@ -36,49 +36,49 @@ interface FetchParameters {
 
 export default class Dump extends Command {
   public static description =
-    'Dump the whole content of a particular source in a CSV format.';
+    'Dump the content of one or more sources into a CSV file.';
 
   public static flags = {
     source: flags.string({
       char: 's',
       description:
-        'The name (not the identifier) of the source(s) for which to extract all documents.',
+        'The names (not the identifiers) of the sources from which to get content.',
       helpValue: 'mySourceName',
       required: true,
       multiple: true,
     }),
     pipeline: flags.string({
       char: 'p',
       description:
-        'The name of the query pipeline for which to extract all documents. If not specified, the default query pipeline will be used.',
+        'The name of the query pipeline through which to get content. If not specified, the default query pipeline is used.',
     }),
     fieldsToExclude: flags.string({
       char: 'x',
       description:
-        'The fields to exclude from the datadump. If not specified, all fields will be returned',
+        'The fields to exclude from the data dump. If not specified, all fields are included.',
       multiple: true,
     }),
     destination: flags.string({
       char: 'd',
       description:
-        'The folder destination where the CSV file should be created',
+        'The folder in which to create the CSV file.',
       default: '.',
     }),
     name: flags.string({
       char: 'n',
-      description: 'The name of the CSV file that should be created',
+      description: 'The name of the CSV file to create.',
       default: 'indexdump',
     }),
     additionalFilter: flags.string({
       char: 'f',
       description:
-        'Additional search filter that should be applied while doing the extraction. See https://docs.coveo.com/en/1552 for more information',
+        'The additional search filter to apply while getting the content. See <https://docs.coveo.com/en/1552>.',
       default: '',
     }),
     chunkSize: flags.integer({
       char: 'c',
       description:
-        'The maximum number of results extract into each CSV file. Default is 10000',
```

Comment:
The current description seems to imply that more than one CSV file could be created if more results are returned than the specified chunkSize. However, other descriptions seemed to imply that only one CSV file would be created for the data dump.

Can more than one file be created? If so, I'll change the documentation a bit.

Reply:
Yes, it will create one file per chunk.